src/main/resources/data/alaskanativecraft/worldgen/configured_structure_feature/configured_fish_camp.json

```diff
@@ -0,0 +1,10 @@
+{
+  "config": {
+    "start_pool": "alaskanativecraft:fish_camp/base",
+    "size": 3
+  },
+  "biomes": "#minecraft:is_beach",
+  "adapt_noise": true,
+  "spawn_overrides": {},
+  "type": "minecraft:village"
+}
```

Comment:
I've got structures working locally, so you can ignore everything structure related for now